js/ui/camera.js

```diff
@@ -411,7 +413,8 @@ class Camera extends Evented {
      * details not specified in `options`.
      *
      * @memberof Map#
-     * @param {CameraOptions|AnimationOptions} options Options describing the destination and animation of the transition.
+     * @param {Object} options Options describing the destination and animation of the transition.
+    *            Accepts [CameraOptions](#CameraOptions) and [AnimationOptions](#AnimationOptions).
```

Comment:
In Flow, our type definition standard, the `|` operator (the "union" operator) indicates that an object may "share attributes from both: https://flowtype.org/docs/union-intersection-types.html#_